client/src/components/Leaflet.js

```diff
@@ -55,54 +54,27 @@ class BaseLeaflet extends Component {
 		return 'map-' + mapId
 	}
 
-	componentDidMount() {
-		let map = L.map(this.mapId, {zoomControl:true}).setView([34.52, 69.16], 10)
-/*
-		let nexrad = L.tileLayer.wms("http://mesonet.agron.iastate.edu/cgi-bin/wms/nexrad/n0r.cgi", {
-		    layers: 'nexrad-n0r-900913',
-		    format: 'image/png',
-		    transparent: true,
-		    attribution: "Weather data © 2012 IEM Nexrad"
-		});
-		let nmra = L.tileLayer.wms("https://mrdata.usgs.gov/services/nmra", {
-			layers: 'USNationalMineralAssessment1998',
-			format: 'image/png',
-			transparent: true
-		})
 
-		let osm = L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png');
+	componentDidMount() {		
+		const mapOptions = Object.assign({zoomControl:true}, Settings.imagery.mapOptions, Settings.imagery.mapOptions.crs && { crs: L.CRS[Settings.imagery.mapOptions.crs] })
+		const map = L.map(this.mapId, mapOptions).setView([34.52, 69.16], 10)
```

Comment:
Maybe nice to put the default center coordinates (34.52, 69.16) and zoom level (10) also in `anet.yml`?

Reply:
Absolutely, great idea!